js/lib/unpackers/p_a_c_k_e_r_unpacker.js

```diff
@@ -80,4 +80,4 @@ var P_A_C_K_E_R = {
     }
 
 
-};
\ No newline at end of file
+};
```

Comment:
I think online editor inserts it: http://i.imgur.com/7bJeVNf.png
Its not binary or text file with very strict structure, so i think its not a big deal.